README.md

```diff
@@ -0,0 +1,9 @@
+#  AWS Serverless Shipper - Lambda
+
+This is an AWS Lambda function that ships logs from AWS serverless apps to Logz.io.
+
+**Note**:
+This project contains code for Python 2 and Python 3.
+We recommend using Python 3 because Python 2 has reached its end of life.
```

Comment:
Will reach EOL at 1.1.20 -- please check this

Reply:
sent you the final version